redesign/index.html

```diff
@@ -15,7 +15,7 @@
     <title>dwyl</title>
   </head>
   <body>
-    <nav class="w-100 vh-100 h3-ns dwyl-bg-dark-grey tc white">
+    <nav class="w-100 vh-100 h3-ns dwyl-bg-dark-grey tc white fixed">
```

Comment:
This will be being removed for the new navbar however, it will be being used in the `menu` class which is on the `ul` like this:
![image](https://user-images.githubusercontent.com/16775804/27130248-70f03ae6-50fd-11e7-98e9-7235eea48f29.png)
This is what I've taken from research resource. I will look into an alternative.